ACLAnthology.js

```diff
@@ -0,0 +1,168 @@
+{
+	"translatorID": "9b2f2d29-02e3-4693-b728-4cd435650669",
+	"label": "ACLAnthology",
+	"creator": "Guy Aglionby",
+	"target": "^https://aclanthology\\.coli\\.uni\\-saarland\\.de/(events|papers|volumes)/[^#]+",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-08-09 10:23:41"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018 Guy Aglionby
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes('/events/') || url.includes('/volumes/')) {
+		return 'multiple';
+	} else if (url.includes('/papers/')) {
+		let id = url.split('/').pop().toLowerCase();
+		return id[0] == 'j' || id[0] == 'q' ? 'journalArticle' : 'conferencePaper';
+	}
+}
+
+function doWeb(doc, url) {
+	const WEBSITE_STUB = 'https://aclanthology.coli.uni-saarland.de/';
+	if (detectWeb(doc, url) === 'multiple') {
+		Zotero.selectItems(extractPapersFromListing(doc), function (selected) {
+			if (!selected) {
+				return true;
+			}
+			
+			Object.keys(selected).forEach(function (id) {
+				let partialBibtexUrl = ZU.xpath(doc, '//p[contains(., "' + id + '")]/a[img[@alt = "Export"]]/@href')[0].value;
+				let bibtexUrl = WEBSITE_STUB + partialBibtexUrl;
+				ZU.doGet(bibtexUrl, function(responseString, responseObj) {
+					scrapeBibtex(responseString);
+				});
+			});
+		});
+	} else if(url.endsWith('.bib')) {
+		scrapeBibtex(ZU.xpath(doc, '//pre')[0].innerHTML);
+	} else {
+		let partialBibtexUrl = ZU.xpath(doc, '//a[contains(., "BibTeX")]/@href')[0].value;
+		let bibtexUrl = WEBSITE_STUB + partialBibtexUrl;
+		ZU.doGet(bibtexUrl, function(responseString, responseObj) {
+			scrapeBibtex(responseString);
+		});
+	}
+}
+
+function extractPapersFromListing(doc) {
```

Comment:
This is fine (except for the author comment below), but would you mind renaming this & following the structure of the `getSearchResults` function we use in templates? (see e.g. https://github.com/zotero/translators/blob/master/Denik%20CZ.js#L53 ) same name&structure for same function makes maintenance so much easier.